Resources/Textures/_Floof/Objects/Fun/floof_plush.rsi/meta.json

```
@@ -90,15 +90,16 @@
         },
         {
             "name": "tiredcargo"
-        },	
+        },
         {
             "name": "unholyred"
-        },	
+        },
         {
             "name": "detective"
-        },	
+        },
         {
             "name": "ursinegal"
+        },
         },	
```

Remove this extraneous bracket.
```suggestion
        },
```